core/src/runtime/api.cpp

```diff
@@ -0,0 +1,7 @@
+#include "api.h"
+
+namespace plc::core::runtime {
+    Result<Ptr> Api::coreVersion() {
```

Comment:
It may be out of scope, but in a wrapper it would be goo to return reference to a C++ struct, so that the user of this class doesn't need to deal with Runtime code convention

Reply:
This will be fixed in my next PR.